spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -571,16 +571,16 @@ function chanUpgradeTry(
   }
 
   // NON CROSSING HELLO CASE:
-  // if the counterparty sequence is less than the current sequence,
+  // if the counterparty sequence is less than or equal to the current sequence,
   // then either the counterparty chain is out-of-sync or the message
   // is out-of-sync and we write an error receipt with our sequence
   // so that the counterparty can abort their attempt and resync with our sequence.
   // When the next upgrade attempt is initiated, both sides will move to a fresh
   // never-before-seen sequence number
   // CROSSING HELLO CASE:
-  // if the counterparty sequence is less than or equal to the current sequence,
+  // if the counterparty sequence is less than the current sequence,
```

Comment:
I think this change is correct because `counterpartyUpgradeSequence` should be `==` to `expectedUpgradeSequence` in the crossing hello scenario.